contracts/Comptroller.sol

```diff
@@ -1008,6 +1008,15 @@ contract Comptroller is Ownable2StepUpgradeable, ComptrollerV1Storage, Comptroll
         return (NO_ERROR, seizeTokens);
     }
 
+    /**
+     * @notice Return all reward distributors for this pool
+     * @return List of RewardDistributor addresses
```

Comment:
I think we should consistent between using array and list. I think we should use `array` because that is what the type is called in solidity
```suggestion
     * @return Array of RewardDistributor addresses
```